README.md

```diff
@@ -1 +1,59 @@
-# conan-recipes-contrib
\ No newline at end of file
+# 🛠️ conan-toolchains
+
+This repository contains Conan recipes for toolchains and command line
+utilities, for example the Emscripten SDK or other compilers. These recipes are
+maintained independently of [Conan Center Index](https://github.com/conan-io/conan-center-index)
+with flexible maintenance scopes.
+
+---
+
+## 🌟 Why a separate repo?
+
+A **centralized** toolchains repository provides a dedicated home for *SDKs*,
+compilers, cross-compilers, and experimental toolchains all in one place. It's
+a great fit for a wide range of users: beginners can easily install a toolchain
+alongside Conan and quickly cross-compile a project, while advanced users can
+integrate it into their workflow by adding it as a local recipe index and
+tailoring recipes as needed. It also supports bleeding-edge or niche toolchains
+that don't belong in the main index, offering maximum flexibility and
+control.
+
+---
+
+## 🚀 Getting started
+
+
+### Setup `conan-toolchains` as a [local recipe index](https://docs.conan.io/2/devops/devops_local_recipes_index.html#devops-local-recipes-index) repository
+
+Before adding `conan-toolchains` ensure [conan](https://github.com/conan-io/conan) is installed and available in your path.
+Check conan [downloads](https://conan.io/downloads) page.
```

Comment:
Link the https://docs.conan.io/2/installation.html better, say you can find instructions for installing there